doc/classes/EditorInterface.xml

```diff
@@ -5,7 +5,7 @@
 	</brief_description>
 	<description>
 		[EditorInterface] gives you control over Godot editor's window. It allows customizing the window, saving and (re-)loading scenes, rendering mesh previews, inspecting and editing resources and objects, and provides access to [EditorSettings], [EditorFileSystem], [EditorResourcePreview], [ScriptEditor], the editor viewport, and information about scenes.
-		[b]Note:[/b] This class shouldn't be instantiated directly. Instead, access the singleton using [method EditorPlugin.get_editor_interface].
+		[b]Note:[/b] This class shouldn't be instantiated directly. Instead, access the singleton directly by its name.
```

Comment:
I wonder if this line could be confusing for C# users, since they need to use the `Singleton` property. But it's a static property so they'll probably find it through IntelliSense.

Reply:
Good point, I'll clarify this.